docs/layouts/index.redirects

```diff
@@ -0,0 +1,11 @@
+{{- $latest  := index site.Data.releases 0 -}}
+{{- $allDocs := where site.RegularPages "Section" "docs" }}
+/docs     /docs/v{{ $latest }}
+/docs/latest     /docs/v{{ $latest }}
```

Comment:
Nit: I noticed that /docs/a/b/c redirects to /docs/v0.10.6/a/b/c but /docs/latest/a/b/c results in 404.

I think that having /docs/latest working but /docs/latest/a/b/c 404-ing will be confusing. WDYT?

We could just remove /docs/latest entirely since /docs/a/b/c seems fine for a floating URL.

Reply:
Good catch. Fixed.